cmd/troubleshoot/dynakube_test.go

```diff
@@ -4,8 +4,10 @@ import (
 	"context"
 	"testing"
 
+	"github.com/Dynatrace/dynatrace-operator/pkg/api/common"
```

Comment:
General Q: why is it common instead of a bunch of smaller packages, each with a different name and purpose?

Reply:
We could do a bunch of small sub packages like:
- `"github.com/Dynatrace/dynatrace-operator/pkg/api/common/image"`
- `"github.com/Dynatrace/dynatrace-operator/pkg/api/common/connectioninfo"`
- ...

But it felt a bit overkill right now, because all the `structs` in it are very simple and small

Reply:
+1

can do, do you have a better name than `common` for the folder (where these little packages will live)?

Reply:
let's think together:
- what do you do:  we create substructs for all APIs? or shared substructs?

Reply:
to be even more percise it's just `structs/` so either substructs or structs is good name IMO
`github.com/Dynatrace/dynatrace-operator/pkg/api/substructs` or `github.com/Dynatrace/dynatrace-operator/pkg/api/structs` or `github.com/Dynatrace/dynatrace-operator/pkg/api/sharedstructs`

Reply:
we could call it `shared`

Reply:
good with me, it's more close what we do there.